apps/expo/app/(app)/(drawer)/(tabs)/(stack)/profile/[id].tsx

```diff
@@ -16,13 +15,13 @@ const Profile = () => {
       {Platform.OS === 'web' && (
         <Head>
           <title>{`${userRealName}'s Profile`}</title>
-          <meta name="description" content={`${userRealName}'s Profile`} />
+          <meta name="description" content={`${id}'s Profile`} />
         </Head>
       )}
       <Stack.Screen
+        name={`${id}'s Profile`}
         options={{
-          title: `${userRealName}'s Profile`,
-          name: `${userRealName}'s Profile`,
+          title: `${id}'s Profile`,
```

Comment:
and this, to be consistent

Reply:
please check it out in this commit [1f17ce0]